netbox/extras/tests/test_event_rules.py

```diff
@@ -376,3 +377,93 @@ def dummy_send(_, request, **kwargs):
         # Patch the Session object with our dummy_send() method, then process the webhook for sending
         with patch.object(Session, 'send', dummy_send) as mock_send:
             send_webhook(**job.kwargs)
+
+    def test_event_rule_conditions_without_logic_operator(self):
```

Comment:
This test case is for testing the processing of event rules. Any tests related to conditions or condition sets specifically should go in `extras.tests.test_conditions.py`. Note that there are two cases in that file: One for individual conditions and one for condition sets.

Reply:
What am I supposed to do with them? Move them to extras.tests.test_conditions.py?